nautobot/docs/development/best-practices.md

```diff
@@ -197,15 +197,9 @@ class UserFilter(NautobotFilterSet):
     - A PK-based filter **should** be made available as well, generally with a name suffixed by `_id`. For example:
```

Comment:
I think this instruction is no longer accurate, given the change to the below example? Please review.